examples/java/core/enumerations/src/main/bond/enumerations.bond

```diff
@@ -0,0 +1,21 @@
+namespace Examples;
+
+enum Color
+{
+    Orange = 1,
+    Yellow = 2,
+}
+
+enum Fruit
+{
+    Apple = 1,
+    Orange = 2,
+}
+
+enum Limits
+{
+    Int32Min = -2147483648,
+    Int32Max = 2147483647,
+    UInt32Min = 0,
+    UInt32Max = 0xFFFFFFFF,
```

Comment:
This example seems to imply that enum values can be used to represent unsigned 32-bit integers, which is not the case since an enum is backed by a signed 32-bit integer.